pkg/api/tests.go

```diff
@@ -50,8 +50,8 @@ const (
 		previous_flakes * 100.0 / NULLIF(previous_runs, 0) AS previous_flake_percentage,
 		(previous_successes + previous_flakes) * 100.0 / NULLIF(previous_runs, 0) AS previous_working_percentage,
 
-		(previous_failures * 100.0 / NULLIF(previous_runs, 0)) - (current_failures * 100.0 / NULLIF(current_runs, 0)) AS net_failure_improvement,
-		(previous_flakes * 100.0 / NULLIF(previous_runs, 0)) - (current_flakes * 100.0 / NULLIF(current_runs, 0)) AS net_flake_improvement,
+		(current_failures * 100.0 / NULLIF(current_runs, 0) - (previous_failures * 100.0 / NULLIF(previous_runs, 0))) AS net_failure_improvement,
+		(current_flakes * 100.0 / NULLIF(current_runs, 0) - (previous_flakes * 100.0 / NULLIF(previous_runs, 0))) AS net_flake_improvement,
```

Comment:
This was reversed, IMHO.  If a test went from a 5% flake rate to a 15% flake, we were returning "-10%" and the UI showed a down arrow.  Now it'll show a red up arrow showing flakiness increased by +10%.

Perhaps `improvement` should be `change` but I'm hesitant to change the name (it dates back all the way to legacy sippy), and it would break URL's.